modules/atomic-opt-in/assets/js/panel-chip/components/popover-card.js

```diff
@@ -62,9 +62,9 @@ const PopoverCard = ( { doClose } ) => {
 					<Button
 						variant="contained"
 						size="small"
-						color="primary"
+						color="accent"
 						onClick={ redirectHandler }
-						sx={ { ml: 'auto' } }
+						sx={ { ml: 'auto', borderRadius: '4px' } }
```

Comment:
In the unstable palette the `borderRadius` value has changed to `8px`. Adar requested me to keep it at `4px`.